addon/components/tinymce-editor.hbs

```diff
@@ -1,7 +1,10 @@
 <div class="tinymce-editor" ...attributes {{did-insert this.initEditor}} {{did-update this.updateValue @value}}>
-    {{#if hasBlock}}
+    {{#if (has-block)}}
         {{yield}}
     {{else}}
-        <textarea id={{this.uniqueId}}></textarea>
+        <textarea id={{this.uniqueId}} style="visibility: hidden;"></textarea>
```

Comment:
I would just throw this style in the addons main .css file (no need to add sass right now), so that it gets added to the app’s vendor.css and is more easily overrideable if you needed to. And to just keep things a little cleaner with no inline styles.